zellij-tile/src/data.rs

```diff
@@ -23,6 +25,31 @@ pub enum Key {
     Esc,
 }
 
+impl Display for Key {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        match self {
+            Key::Backspace => write!(f, "BS"),
```

Comment:
It'd be nice to get some additional feedback on this `Display` implementation! For the most part, I think it's great, but I might actually prefer things like:

- `BS` -> `BkSp`
- `PGUP` -> `PgUp`
- `PGDN` -> `PgDn`
- `BT` -> `BkTb`
- `ALT` -> `Alt`

I have the feeling that is *very* subjective though, so it'd be nice to get some more opinions!

Reply:
Fixed in https://github.com/zellij-org/zellij/pull/319/commits/3e6dcc0ac319e3fca38773f65a97b3fb4e784969